src/main/java/parser/Parser.java

```diff
@@ -39,14 +95,19 @@ private CommandEnum parseTypeOfCommand(String userInput) throws CommandException
             return CommandEnum.LIST;
         case "/view":
             return CommandEnum.VIEW;
+        case "/favourite":
+            return CommandEnum.FAVOURITE;
+        case "/settings":
+            return CommandEnum.SETTINGS;
         default:
             throw new CommandException("Please enter a valid command");
         }
     }
 
     private ArrayList<ArgumentFlagEnum> parseTypeOfArgument(String userInput) {
         ArrayList<ArgumentFlagEnum> argEnumList = new ArrayList<>();
-        Pattern pattern = Pattern.compile(" /[dnilk] ");
+
+        Pattern pattern = Pattern.compile(FLAG_REGEX);
```

Comment:
Perhaps a a better word than pattern for the variable name?

Reply:
alright fixed!